README.md

```diff
@@ -32,15 +45,30 @@ pip install --no-deps -U -e hexrd
 CONDA_BUILD=1 pip install --no-deps -U -e hexrdgui
```

Comment:
We can remove the first python install, only need one!

Reply:
Oops, overlooked that. Fixed.